clang-tidy/action.yaml

```diff
@@ -27,6 +27,12 @@ runs:
         echo "target packages: ${{ inputs.target-packages }}"
       shell: bash
 
+    - name: Install curl
+      run: |
+        sudo apt-get -yqq update
+        sudo apt-get -yqq install curl
```

Comment:
The reason I used `-yqq` is to reduce the log output, but there was no effect. :pleading_face: 

![image](https://user-images.githubusercontent.com/31987104/146017496-98567a52-3d73-4dd9-af29-e3594dec2ab5.png)